addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2145,6 +2145,7 @@ export function setTagName(el, newTagName) {
     }
     const n = document.createElement(newTagName);
     if (el.nodeName !== 'LI') {
+        el.style.removeProperty('list-style');
```

Comment:
A testcase for this would be appreciated.

Reply:
https://github.com/odoo/odoo/pull/156677/files#diff-e71695c509758d0bfe5b57bfd75d18f5c7f489b36c39f1decbf692f8e51488c4R140